src/battle_script_commands.c

```diff
@@ -5155,66 +5155,69 @@ static void atk5C_hitanimation(void)
     }
 }
 
-#ifdef NONMATCHING
 static void atk5D_getmoneyreward(void)
 {
     u32 i = 0;
-    u32 lastMonLevel = 0;
-    u32 moneyReward = 0;
-    u32 value;
+    u32 moneyReward;
+    u8 lastMonLevel = 0;
+    
+    const struct TrainerMonNoItemDefaultMoves *party1;
+
+    const struct TrainerMonNoItemCustomMoves *party2;
+
+    const struct TrainerMonItemDefaultMoves *party3;
+
+    const struct TrainerMonItemCustomMoves *party4;
+
 
     // The whole function is using wrong registers. 
```

Comment:
If it's matched now, do you still need this comment?

Reply:
No